apps/sports-helsinki/src/domain/venue/venueCard/LargeVenueCard.tsx

```diff
@@ -67,7 +69,15 @@ const LargeVenueCard: React.FC<Props> = ({
       >
         <div className={styles.eventCard}>
           <div className={styles.infoWrapper}>
-            <div className={styles.eventName}>{title}</div>
+            <div className={styles.eventName}>
+              {title}
+              {isHelsinkiCityOwned && (
+                <IconCheckCircleFill
+                  className={styles.helsinkiCityOwnedIcon}
+                  aria-hidden
+                />
```

Comment:
Should this have some kind of tooltip that would tell what this is for?

Reply:
I talked with the designer about this and he approved this feature without tooltips. It would be ideal to have the tooltips in the detail page at least, in the search cards it was discussed that it might have to be tried out how it worked because the cards themselves are clickable and using mobile one has to click to get the tooltip shown and thus there would be two clickable actions in the card according to where one clicked. So, ideally yes for the detail page and maybe for the search result cards. As the tooltip functionality is not required I'll probably leave it out of the scope of this PR.